src/sentry/integrations/utils/code_mapping.py

```diff
@@ -151,13 +148,13 @@ def list_file_matches(self, frame_filename: FrameFilename) -> List[CodeMappingMa
 
             for file in matches:
                 file_matches.append(
-                    CodeMappingMatch(
-                        filename=file,
-                        repo_name=repo_tree.repo.name,
-                        repo_branch=repo_tree.repo.branch,
-                        stacktrace_root=f"{frame_filename.root}/",
-                        source_path=self._get_code_mapping_source_path(file, frame_filename),
-                    )
+                    {
+                        "filename": file,
+                        "repo_name": repo_tree.repo.name,
+                        "repo_branch": repo_tree.repo.branch,
+                        "stacktrace_root": f"{frame_filename.root}/",
+                        "source_path": self._get_code_mapping_source_path(file, frame_filename),
+                    }
```

Comment:
I can see now more easily why you were tying to use the CodeMappingMatch class. My apologies.
If you're happy with the current implementation go with it.

Alternative, you could append tuples like this: `(file, CodeMapping)`.

Reply:
I think the dictionary is okay - we won't be reusing this in any other places so I get why it's not worth defining a specific class.